.changeset/gentle-buses-walk.md

```diff
@@ -0,0 +1,5 @@
+---
+"sku": patch
+---
+
+Update eslint-config-seek to version 12.
```

Comment:
IMO it would be good to have a message here mentioning that the majority of changes should be autofixable.